code/go/0chain.net/smartcontract/storagesc/allocation.go

```diff
@@ -500,16 +490,14 @@ func (uar *updateAllocationRequest) decode(b []byte) error {
 func (uar *updateAllocationRequest) validate(
 	conf *Config,
 	alloc *StorageAllocation,
-) (err error) {
+) error {
 	if uar.SetImmutable && alloc.IsImmutable {
```

Comment:
Is it possible to have uar.SetImmutable = false, and alloc.IsImmutable is true, i.e, try to unset the immutable.

Reply:
I believe immutable status is permanent. `uar.SetImmutable = false means we are not doing anything regarding imutablity.